android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/extensions/CameraPadding.kt

```diff
@@ -0,0 +1,29 @@
+package com.stadiamaps.ferrostar.maplibreui.extensions
+
+import com.maplibre.compose.camera.models.CameraPadding
+
+/**
+ * A camera padding configuration for a navigation map view that places the user puck in the lower
+ * center of the screen.
+ */
+fun CameraPadding.Companion.NavigationPortrait(): CameraPadding {
```

Comment:
General Android lint I noticed... I know what you're going for here though ;) Not sure if it's the right way or not TBH. I won't block the PR for this but something to think about.

<img width="907" alt="image" src="https://github.com/user-attachments/assets/4d6787cb-d8d8-4b3f-91f7-9a97e1af8594">

Reply:
Yep. I've been ignoring those to keep it "kotlin enum like". I could put an actual suppression on it to note it's an intentional decision.